form.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PosterShop</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600&display=swap" rel="stylesheet">
+
+
+</head>
+<body>
+
+ <nav>
+  <h2>PosterShop</h2>
+    <div class="nav-links">
+      <a href="index.html">Home</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+    </div>
+ </nav>
+
+ <section class="signup-form">
+    <h2>Join our Newsletter</h2>
+    <p>Stay up to date with new posters and exclusive deals</p>
+    <form action="https://httpbin.org/anything" method="post">
+
+      <div>
+        <label for="firstname">Firstname</label>
+        <input id="firstname" type="text" name="name" placeholder="Firstname" required>
+      </div>
+
+      <div>
+        <label for="lastname">Lastname</label>
+        <input id = "lastname" type="text" name="lastname" placeholder="Lastname" required>
```

Comment:
There shouldn't be any spaces around the equal sign. And please indent your attributes going forward so that it's more readable.